src/dune/pform.ml

```diff
@@ -241,6 +241,13 @@ module Map = struct
       let model = Ocaml_config.model context.ocaml_config in
       let system = Ocaml_config.system context.ocaml_config in
       let version_string = Ocaml_config.version_string context.ocaml_config in
+      let ext_plugin =
+        Mode.plugin_ext
+          ( if Ocaml_config.natdynlink_supported context.ocaml_config then
```

Comment:
We have this `Dynlink_supported` module that's supposed to be used for making these checks. I don't quite remember what its purpose is but it's worth looking at to make sure we aren't missing anything.

Reply:
I just looked, I think it is OK like this. The `Dynlink_supported` module seems to be used to handle `(no_dynlink)`. I could use `Dynlink_supported.By_the_os.get` instead, but it amounts to the same thing, and I think it ends up obfuscating things more than anything else.